packages/web-components/src/components/leadspace/leadspace-body.ts

```diff
@@ -0,0 +1,182 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, property, LitElement } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import styles from './leadspace.scss';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Leadspace type
+ */
+export enum LEADSPACE_TYPE {
+  /**
+   * Left-aligned - small style of the leadspace title (default)
+   */
+  SMALL = 'small',
+
+  /**
+   * Left-aligned - large style of the leadspace title
+   */
+  LEFT = 'left',
+
+  /**
+   * Centered type of the LeadSpace
+   */
+  CENTERED = 'centered',
+}
+
+/**
+ * Leadspace theme
+ */
+export enum LEADSPACE_THEME {
+  /**
+   * Carbon White theme (default)
+   */
+  WHITE = 'white',
+
+  /**
+   *  Carbon Gray 100 theme
+   */
+  G100 = 'g100',
+}
+
+/**
+ * LeadSpace body.
+ */
+abstract class DDSLeadSpaceBody extends LitElement {
```

Comment:
Naming nit, from the `abstract` annotation and how this class is used:

```suggestion
abstract class DDSLeadSpaceBase extends LitElement {
```